includes/blocks/class-wc-block-featured-product.php

```diff
@@ -29,6 +29,7 @@ class WC_Block_Featured_Product {
 		'align'        => 'none',
 		'contentAlign' => 'center',
 		'dimRatio'     => 50,
+		'height'       => 500,
```

Comment:
Would be awesome if the 500 was set here in the PHP, then passed through the block settings and used from there. Then it could be hooked and changed if needed.